scripts/myMangaDex.js

```diff
@@ -43,7 +43,7 @@ class MyMangaDex {
 
 		// Choose page
 		if ((this.pageUrl.indexOf(urls.follows) > -1 && this.pageUrl.indexOf("/manga/") == -1) ||
-			(this.pageUrl.indexOf(urls.group) > -1 && (this.pageUrl.indexOf("/chapters/") > -1 || (this.pageUrl.indexOf("/manga/") == -1 && this.pageUrl.indexOf("/comments/") == -1))) ||
+			(this.pageUrl.indexOf(urls.group) > -1 && (this.pageUrl.indexOf("/chapters/") > -1 || (this.pageUrl.indexOf("/manga/") == -1 && this.pageUrl.indexOf("/comments") == -1))) ||
```

Comment:
I don't think this is necessary ? Did the ``/comments`` page trigerred the extension ?
This conditions makes it so only ``/chapters`` validate the condition, and both ``/chapters`` and ``/comments`` can't be in the URL at the same time.

Reply:
I got the update notification on the comments page, adding this stopped it

Reply:
Sorry I misread the change, yes there is an useless ``/`` char.